src/pages/comparison/comparison.tsx

```diff
@@ -175,11 +196,32 @@ const RightLayersPanelWrapper = styled(LeftLayersPanelWrapper)`
   })};
 `;
 
+const getLayerExamples = (): LayerExample[] => Object.values(EXAMPLES);
+
 export const Comparison = ({ mode }: ComparisonPageProps) => {
   let currentViewport: WebMercatorViewport = null;
+
+  const [leftSideExamples, setLeftSideExamples] = useState<LayerExample[]>(() =>
+    getLayerExamples()
+  );
+
```

Comment:
reduce number of empty lines here. This is ordinary piece of code. Don't need to highlight every hook.

Reply:
Done